target/linux/x86/patches-4.19/010-no_stack_validation.patch

```diff
@@ -0,0 +1,12 @@
+--- a/arch/x86/Kconfig		2020-03-12 20:01:25.012257298 -0400
++++ b/arch/x86/Kconfig		2020-03-12 20:01:52.716024243 -0400
+@@ -185,7 +185,8 @@
+ 	select HAVE_REGS_AND_STACK_ACCESS_API
+ 	select HAVE_RELIABLE_STACKTRACE		if X86_64 && (UNWINDER_FRAME_POINTER || UNWINDER_ORC) && STACK_VALIDATION
+ 	select HAVE_STACKPROTECTOR		if CC_HAS_SANE_STACKPROTECTOR
+-	select HAVE_STACK_VALIDATION		if X86_64
++# this requires objtool
```

Comment:
If that option is is causing issues, then it should be removed. Git commit description is enough, no need to add another comment to the patch.